src/js/entities-service/entities/patients.js

```diff
@@ -48,7 +48,7 @@ const _Model = BaseModel.extend({
     attrs = extend({}, this.attributes, attrs);
 
     const relationships = {
-      'groups': this.toRelation(attrs._groups, 'groups'),
+      'groups': this.toRelation(attrs._groups, 'workspaces'),
```

Comment:
Just FYI, the relationships _should_ have been duplicated in the API too, so anything that exists in `relationships.group` or `relationships.groups` would also be `relationships.workspace` (s).